doc/api/crypto.markdown

```diff
@@ -627,17 +627,26 @@ You can get a list of supported digest functions with
 
 Synchronous PBKDF2 function.  Returns derivedKey or throws error.
 
-## crypto.randomBytes(size[, callback])
+## crypto.randomBytes(size, callback)
 
 Generates cryptographically strong pseudo-random data. Usage:
 
-    // async
     crypto.randomBytes(256, function(ex, buf) {
       if (ex) throw ex;
       console.log('Have %d bytes of random data: %s', buf.length, buf);
     });
 
-    // sync
+NOTE: This will block if there is insufficient entropy, although it should
```

Comment:
no. i didn't change any of the copy. it just shows up in the diff because code moved around.